packages/react-components/react-combobox/etc/react-combobox.api.md

```diff
@@ -81,7 +81,7 @@ export type DropdownOpenChangeData = ComboboxBaseOpenChangeData;
 export type DropdownOpenEvents = ComboboxBaseOpenEvents;
 
 // @public
-export type DropdownProps = ComponentProps<Partial<DropdownSlots>, 'button'> & ComboboxBaseProps;
+export type DropdownProps = ComponentProps<Partial<DropdownSlots>, 'button'> & Omit<ComboboxBaseProps, 'freeform'>;
```

Comment:
Rather than have this `Omit<>`, I think it would make more sense to only add `freeform` to `ComboboxProps`. That way `ComboboxBaseProps` can stay a type that only contains the shared props between Combobox & Dropdown.

It looks like the picker types are pulling directly from `ComboboxProps`/`ComboboxState`, so I don't think this should make it harder to use there. Let me know if there's another reason to have this here, though.

Reply:
Yeah, I've already removed `freeform` from `ComboboxBaseProps` actually, so this `Omit` is quite useless, I'll remove it too!

Reply:
I want to expand the usage of `ComboboxBaseProps` and `ComboboxBaseState` to be used by `TagPicker` also, as `TagPicker` shares most of the logic of `Combobox` but without sharing combobox's slots. 

That is why adding both `freeform` kind of makes sense, as those logics were basically repeating between `Combobox` and `TagPicker` it seems like the logic can be shared. `Dropdown` can opt-out of this logic by simply omitting `freeform` (and providing it internally as `false`)

Reply:
Ok, I've remove the omission and also removed `freeform` from `ComboboxBaseProps ` but kept it on `ComboboxBaseState` and `useComboboxBaseState` signature, as `freeform` is a shared logic between `Combobox` and `TagPicker`, the `Dropdown` is opting-out of this logic by not providing a `freeform` property (and also by providing `false` on the hook invocation)